topics/brave/index.md

```diff
@@ -0,0 +1,15 @@
+---
+aliases: brave-browser
+related: chrome,firefox,safari,edge,opera,chromium,browser
+created_by: Brave Software
+display_name: Brave Browser
+github_url: https://github.com/brave/brave-browser
+logo: brave.png
+released: November 13, 2019
+short_description: Brave is an open source web browser from Brave Software.
+topic: brave
+url: https://www.brave.com
+wikipedia_url: https://en.wikipedia.org/wiki/Brave_(web_browser)
+---
+
+Brave is a free and open source web browser developed by Brave Software. It is a privacy-focused browser based on the open source Chroimum project. It's available for most operating systems including Windows, macOS, Linux and most phones and tablets.

```

Comment:
Spelling fix on "Chromium":

```suggestion
Brave is a free and open source web browser developed by Brave Software. It is a privacy-focused browser based on the open source Chromium project. It's available for most operating systems including Windows, macOS, Linux and most phones and tablets.
```

Reply:
thanks i am new on in these
you can helpp me i whant cancel the transfer from starting and my money back in my adress